packages/components/src/ui/v-stack/README.md

```diff
@@ -0,0 +1,123 @@
+# VStack
+
+`VStack` (or Vertical Stack) is a layout component that arranges child elements in a vertical line.
+
+## Usage
+
+`VStack` can render anything inside.
+
+```jsx live
+import { VStack, Text, View } from '@wp-g2/components';
+import { ui } from '@wp-g2/styles';
+
+function Example() {
+	return (
+		<VStack css={[ui.frame.height(200)]}>
```

Comment:
I think we discussed avoiding these css props in public components right? what's the status of that?

Reply:
It's a good point, we should avoid it in the examples as well. I'll make a note to clean these up as we go along

cc/ @ItsJonQ

Reply:
@sarayourfriend Good call. We'll remove it from the examples